third_party/2and3/click/core.pyi

```diff
@@ -313,6 +313,10 @@ class CommandCollection(MultiCommand):
         ...
 
 
+# This type is here to resolve https://github.com/python/mypy/issues/5275
+_ConvertibleType = Union[type, ParamType, Tuple[type, ...], Callable[[str], Any], Callable[[Optional[str]], Any]]
```

Comment:
Could you put this alias in `utils.pyi` so that you can avoid both an import cycle and the `Any` you added in `types.pyi`?

Reply:
I don't think that would fix the import cycle. It would just punt it down the road (utils would need to import types, types would need to import utils).

Reply:
Oh right, I didn't realize that this definition depends on ParamType.

In that case, we can get around this by moving ParamType into utils.pyi, prefixed with an underscore to make it private. In types.pyi, we can then `from .utils import _ParamType as ParamType`.

Reply:
That seems not to fix it either. Based on the minmal repro I just found, I think Parameter and ParamType need to be in the same file. I am about to push something that passes (if you would like me to move things around, while keeping these two classes in the same file, let me know).